ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -555,6 +656,8 @@ def get_partial_event_contingency_tables(self) -> Tuple[List[EventContingencyTab
         query, params = self.get_contingency_table_query()
         results_with_total = sync_execute(query, params)
 
+        assert isinstance(results_with_total, list)
```

Comment:
Is this supposed to be here? I think the result from sync_execute should be consistent and if not we should handle it in a way that won't just error if it fails

Reply:
have removed